MustangLib-Imported/src/main/java/frc/team670/mustanglib/commands/MustangCommand.java

```diff
@@ -0,0 +1,23 @@
+package frc.team670.mustanglib.commands;
+
+import frc.team670.mustanglib.subsystems.MustangSubsystemBase;
+
+import java.util.Map;
+
+import edu.wpi.first.wpilibj.controller.PIDController;
+import edu.wpi.first.wpilibj.controller.RamseteController;
+import edu.wpi.first.wpilibj.controller.SimpleMotorFeedforward;
+
+/**
+ * Represents a robot action with defined health requirements for every subsystem it uses.
+ * 
+ * @author ctychen, lakshbhambhani
+ */
+public interface MustangCommand{
```

Comment:
we already had a MustangCommand, right? I'm wondering why this is a new file and wasn't the old file being moved